src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -222,25 +222,13 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
     return out
   }, [currencyCode, currencyConfigMap])
 
-  const edgeStakingAssets =
-    allStakePolicies == null
-      ? []
-      : matchingEdgeAssets.filter(
-          asset => filterStakePolicies(allStakePolicies, { pluginId: asset.pluginId, currencyCode: currencyCode.toUpperCase() }).length > 0
-        )
-
-  /** Check if all the stake plugins are loaded for this asset type */
-  const isStakingLoading =
-    stakingWallets.length === 0 ||
-    stakingWallets.every(
-      wallet =>
-        walletStakingStateMap[wallet.id] == null ||
-        walletStakingStateMap[wallet.id].stakePlugins.length === 0 ||
-        Object.keys(walletStakingStateMap[wallet.id].stakePolicies).length === 0
-    ) ||
-    edgeStakingAssets.length === 0 ||
-    allStakePolicies == null ||
-    allStakePolicies.length === 0
+  const edgeStakingAssets = matchingEdgeAssets.filter(
+    asset => filterStakePolicies(allStakePolicies, { pluginId: asset.pluginId, currencyCode: currencyCode.toUpperCase() }).length > 0
+  )
+
+  /** Check if all the stake plugins/policies are loaded for this asset type */
```

Comment:
Since we are going with `/** */`, this comment describes the following variable, not the following code. So should be "True if this currency supports staking"